src/components/Download/DatasetDetails/ExperimentView/ViewBlock.js

```diff
@@ -13,14 +13,13 @@ import { TextNull } from 'components/shared/TextNull'
 import { ViewSamplesButton } from '../ViewSamplesButton'
 
 export const ViewBlock = ({
-  datasetId,
-  addedSamples,
+  dataset,
```

Comment:
I've passed the entire data structure `dataset` to this component to match the refactored pattern.